HISTORY.md

```diff
@@ -828,59 +828,59 @@
 * chore(common/resources): bump @actions/http-client from 1.0.3 to 1.0.8 in /resources/build/version (#3047)
 * fix(oem/fv/ios): firstvoices icon and version info (#3044)
 * feat(windows): Sentry integration fixes and polish (#3006)
-* feat(common/lmlayer): compile joinWordsAt property (#3032)
+* feat(developer/compilers): compile joinWordsAt property (#3032)
 
 ## 14.0.60 alpha 2020-04-29
 
-* feat(common): initial use of lerna (in-repo package links only) (#2997)
+* feat(common/resources): initial use of lerna (in-repo package links only) (#2997)
 * change(web/engine): spins core/web/keyboard-processor package off from KeymanWeb (#3001)
 * change(web/engine): spins core/web/input-processor package off from KeymanWeb (#3008)
-* chore(common/lmlayer): change author from personal to work affiliation (#3046)
+* chore(common/models): change author from personal to work affiliation (#3046)
 
 ## 14.0.59 alpha 2020-04-29
 
 * chore: merge stable history (#3037)
-* feat(common/lmlayer): create join word breaker decorator (#3021)
+* feat(common/models/wordbreakers): create join word breaker decorator (#3021)
 * fix(oem/fv/ios): download fv keyboards (#3040)
 * change(android): Refactor LanguageResource() and remove "Custom" property (#3033)
 
 ## 14.0.58 alpha 2020-04-28
 
-* change(common/lmlayer): improvements to default searchTermToKey (#3024)
+* change(developer/compilers): improvements to default searchTermToKey (#3024)
 
 ## 14.0.57 alpha 2020-04-27
 
-* change(common): Update to Unicode 13.0 (#3029)
+* change(common/resources): Update to Unicode 13.0 (#3029)
 * change(android): Refactor Keyboard class to not use Map (#3020)
 
 ## 14.0.56 alpha 2020-04-24
 
 * fix(android): Clarify label that shows "Get Started" on startup (#3025)
-* feat(common/lmlayer): allow for verbose word breaker specification (#3023)
+* feat(developer/compilers): allow for verbose word breaker specification (#3023)
 
 ## 14.0.55 alpha 2020-04-23
 
 * change(android): Convert LanguageListActivity to utility (#3018)
 * refactor(web/engine): successful web-core compilation (#2992)
-* change(common/lmlayer): always bundle searchTermToKey() with model (#2971)
-* change(common/lmlayer): remove NFD table (#3014)
+* change(common/models/developer/compilers): always bundle searchTermToKey() with model (#2971)
```

Comment:
```suggestion
* change(common/models, developer/compilers): always bundle searchTermToKey() with model (#2971)
```

- [ ] Required for my approval.  (Or a similar variant; pretty sure this was a typo.)